### DISCUSSION TITLE
Readering using the sort field with drag-and-drop should update the `date_updated` column.

### DISCUSSION THREAD

**Discussion by LLBlumire** [directus/directus#5253] [Feature Requests]
I am writing an application which queries the Directus API for all items in a collection that have a `date_created` or `date_updated` that is after the last time they were requested (it is then stored in IndexedDB on the client browser). This allows the user to only download new data on application load, rather than having to redownload the entire data set each time.

With this, I noticed that re-ordering items in a collection using the drag and drop (which updates the `sort` field, in my case I have set that to a custom field called `number` as these are lesson numbers for a virtual teaching application) does not cause the `date_updated` column to change. Thus this information is never synced to my clients.

I am not sure if this would be counted as a bug, or a new feature, and Ben from the core team advised I open this discussion here.

**Comment by rijkvanzanten** [comment]
By updating https://github.com/directus/directus/blob/63c45a6ae4cbfa20064808d4002e2f5fc8269a9f/api/src/services/utils.ts#L18 to use the ItemsService.updateByQuery instead of going straight to DB, this should be resolved

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
